packs/data/feats.db/marshal-dedication.json

```diff
@@ -44,6 +44,96 @@
                     "mental",
                     "visual"
                 ]
+            },
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "PF2E.SkillDip",
+                        "predicate": [
+                            "skill:dip:rank:0"
+                        ],
+                        "value": "dip-trained"
+                    },
+                    {
+                        "label": "PF2E.SkillDip",
+                        "predicate": [
+                            "skill:dip:rank:1"
+                        ],
+                        "value": "dip-expert"
+                    },
+                    {
+                        "label": "PF2E.SkillItm",
+                        "predicate": [
+                            "skill:itm:rank:0"
+                        ],
+                        "value": "itm-trained"
+                    },
+                    {
+                        "label": "PF2E.SkillItm",
+                        "predicate": [
+                            "skill:itm:rank:1"
+                        ],
+                        "value": "itm-expert"
+                    }
+                ],
```

Comment:
This is going to fail out the entire choice set and then the rest of the rule elements on the item if they're expert or better on both diplomacy and intimidation.